packages/hooks/src/useCFDDemoAccounts.ts

```diff
@@ -1,13 +1,20 @@
 import useCFDAllAccounts from './useCFDAllAccounts';
+import useExchangeRate from './useExchangeRate';
 
 /**
  *  we can use this hook to get the CFD demo accounts.
  *  it loops through the all of user's CFD accounts, finds and returns demo accounts
  */
 const useCFDDemoAccounts = () => {
+    const { getRate } = useExchangeRate();
     const cfd_accounts = useCFDAllAccounts();
 
-    const cfd_demo_accounts = cfd_accounts.filter(account => account.account_type === 'demo');
+    const cfd_demo_accounts = cfd_accounts
+        .filter(account => account.account_type === 'demo')
+        .map(account => ({
+            ...account,
+            balance: account.balance ? account.balance / getRate(account.currency || '') : account.balance,
```

Comment:
```suggestion
            balance: account.balance && account.currency  ? account.balance / getRate(account.currency) : account.balance,
```

@keoy, why not return `balance` as is if the `account` has no `currency`? 🤔